_includes/references/manifest-web-settings.md

```diff
@@ -572,8 +572,8 @@ Specifies whether your servers should communicate over <code>private</code> or <
   <tr>
     <td><code>vpc_id</code></td>
     <td><div class="tooltip">Build-only &#9432;<span class="tooltiptext">This setting only applies when the app is first built (or cloned) or when new servers are added.</span></div></td>
-    <td><b>ID</b> or <b>name</b> of the AWS VPC in which you would like to create your servers. Note that you must provide subnet_id for all servers in your application.</td>
-    <td>AWS</td>
+    <td><b>ID</b> or <b>name</b> of the VPC in which you would like to create your servers. Note that you must provide subnet_id for all servers in your application.</td>
+    <td>AWS, Hetzner</td>
```

Comment:
I'm not sure if subnet_id is applicable to Hetzner, I haven't come across it during testing. I was able to specify a vpc_id without specifying a subnet_id.

Reply:
@oppchri ok, sorted - have made it AWS-specific